src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public static String getRandomColor() {
+        Colors[] color = Colors.values();
+        int index = new Random().nextInt(color.length);
+        return color[index].toString();
```

Comment:
Please read common mistakes more cerafully (https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)